src/overview.md

```diff
@@ -183,27 +183,68 @@ To find additional APIs, see the
 [commonly used packages page](/guides/libraries/useful-libraries).
 
 
-## Dart: The platform
+## Dart: The platform {#platform}
```

Comment:
If we're going to talk about "Native platform" and "Web platform" below, should this be "Dart: The platforms"?